app/assets/stylesheets/articles.scss

```diff
@@ -111,11 +111,7 @@
             theme-container-accent-background,
             lighten($bold-blue, 8%)
           );
-          @include themeable(
```

Comment:
Aside from the comment at the end of this file (the real change), the rest is this just being formatted correctly.